tutorials/avail-da.md

```diff
@@ -0,0 +1,106 @@
+# GM world rollup: Deploying to Avail  
+
+## 🌞 Introduction {#introduction}
+
+Avail DA offers scalable data availability that underpins the Avail ecosystem and ensures instantaneous and reliable data integrity, enabling rollups to grow, through the use of cutting-edge zero knowledge and KZG Polynomial commitments.
+
+This tutorial serves as a comprehensive guide for deploying your GM world rollup on Avail's data availability (DA) network.
+
+Before proceeding, ensure that you have completed the [GM world rollup](/tutorials/gm-world) tutorial, which covers setting up a local sovereign gm-world rollup and connecting it to a local (mock) DA node.
+
+## 🪶 Running an avail light node
+
+Before you can start your rollup node, you need to initiate, sync, and possibly fund a light node on Turing testnet which is the test network of Avail
+
+- [Local Development](https://github.com/rollkit/avail-da/blob/main/README.md)
+- [Turing Testnet](https://docs.availproject.org/docs/networks)
+
+### 🚀 Using Turing Testnet
+
+- To fund your wallet address for using Turing Testnet: get AVAIL tokens from [Faucet]((https://faucet.avail.tools/))
+- Paste your mnemonic in the `identity.toml` file by creating a `identity.toml` with the following command:
+```touch identity.toml```
+Example:
+```bash
```

Comment:
- [ ] add spaces around code blocks for linting, in all instances in this document